18 ReactHotLoader/package.json

```diff
@@ -41,14 +43,17 @@
     "karma-webpack": "^1.7.0",
     "mocha": "^2.4.5",
     "react-addons-test-utils": "^15.2.1",
+    "react-hot-loader": "^1.3.0",
     "redux-mock-store": "^1.1.2",
+    "rimraf": "^2.5.4",
     "sinon": "^1.17.3",
     "style-loader": "^0.13.0",
     "ts-loader": "^0.8.1",
     "tsd": "^0.6.5",
     "typescript": "^1.8.9",
     "url-loader": "^0.5.7",
     "webpack": "^1.12.13",
-    "webpack-dev-server": "~1.10.1"
+    "webpack-dev-server": "^1.14.1",
+    "webpack-hot-middleware": "^2.12.2"
```

Comment:
[webpack-hot-middleware](https://github.com/glenjamin/webpack-hot-middleware) is used when we are not using webpack-dev-server